src/openmrs-esm-login.tsx

```diff
@@ -3,6 +3,14 @@ import React from "react";
 import ReactDOM from "react-dom";
 import singleSpaReact from "single-spa-react";
 import Root from "./root.component";
+// import { loadConfig } from "@openmrs/esm-root-config";
```

Comment:
Is this dead code? Should it be removed?

Reply:
Yep